_content/applications/tago/index.md

```diff
@@ -4,36 +4,47 @@ section: Integrations
 source: 'https://github.com/TheThingsNetwork/docs/blob/master/_content/applications/tago/index.md'
 ---
 
-# Tago
+# TagoIO
 
-[Tago](https://tago.io/) allows you to quickly store, visualize, and act on sensor data.
+TagoIO is the best IoT platform to build nice dashboards, analytics, reports, and notifications easily and quickly. Also, you can manage devices and users, and add your own logo, colors, and url to deploy your solution in minutes.
```

Comment:
There's no room for marketing in our docs. Please revert to more neutral wording